tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -15,6 +15,21 @@ build --action_env=CACHEBUSTER=565341047
 # Build options for GPU Linux
 build --config=release_gpu_linux
```

Comment:
This line pulls in the cuda config from the .bazelrc.
We'll need a "release_rocm_linux" config there I think..